generateRandomQueries/README.md

```diff
@@ -0,0 +1,2 @@
+In order to generate a sql file that contains random procedures and calls, use `generate_procedure.py`. 
+This will generate `init.sql` and `query.sql`. `init.sql` contains table definitions, settings, procedures etc. `query.sql` contains producedure calls.
```

Comment:
Minor: Maybe mention that this file is used to generate procedures that are pushed down to the workers (uses function call delegation )